atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -10,9 +10,17 @@
 try:
     # py2
     from ConfigParser import ConfigParser
+    # Although io.StringIO is available in python2, it
+    # always expects unicode during write. Since
+    # ConfigParser.ConfigParser does not support
+    # writing unicode, io cannot be used.
+    from StringIO import StringIO
```

Comment:
I tried that, but it seems to not work if `from __future__ import unicode literals` is used.